test/e2e/fleet_test.go

```diff
@@ -869,7 +867,7 @@ func TestFleetNameValidation(t *testing.T) {
 	require.NotNil(t, err)
 	statusErr := err.(*k8serrors.StatusError)
 	assert.True(t, len(statusErr.Status().Details.Causes) > 0)
-	assert.Equal(t, metav1.CauseTypeFieldValueInvalid, statusErr.Status().Details.Causes[0].Type)
+	assert.Equal(t, metav1.CauseType("FieldValueTooLong"), statusErr.Status().Details.Causes[0].Type)
```

Comment:
This field is not yet available in metav1, so this direct type conversion

Reply:
https://github.com/kubernetes/kubernetes/issues/119152

I asked this question at kubernetes